### PR TITLE
Fix issues preventing glean probes from being visualized (dev)

### PR DESCRIPTION
**Pull request by edugfilho**
Most issues derived from the assumption glean had non normalized data, but it doesn't.

### CODE REVIEW

File: src/components/explore/QuantileExplorerView.svelte [hunk 1]
```diff
@@ -173,9 +173,6 @@
               comparisonKeyFormatter={(perc) => `${perc}%`}
               yScaleType={probeType === 'log' ? 'scalePoint' : 'linear'}
               {yDomain}
-              )
-              ),
-              ]}
```
Comment (Iinh) [inline src/components/explore/QuantileExplorerView.svelte:178]: @edugfilho why was this removed?

Comment (edugfilho) [reply src/components/explore/QuantileExplorerView.svelte:178]: Seems like this was leftover from another PR. None of those characters were closing anything. A local lint error on line 177 drew my attention to this.

**Review by Iinh — APPROVED** [review]
